dotCMS/src/main/java/com/dotcms/workflow/form/WorkflowReorderActionStepForm.java

```diff
@@ -0,0 +1,79 @@
+package com.dotcms.workflow.form;
+
+import com.dotcms.repackage.com.fasterxml.jackson.annotation.JsonProperty;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.dotcms.rest.api.Validated;
+
+@JsonDeserialize(builder = WorkflowReorderActionStepForm.Builder.class)
+public class WorkflowReorderActionStepForm extends Validated {
+
+    @NotNull
+    private final String        actionId;
+
+    @NotNull
+    private final String        stepId;
+
+    @NotNull
+    private final int           order;
+
+    public String getActionId() {
+        return actionId;
+    }
+
+    public String getStepId() {
+        return stepId;
+    }
+
+    public int getOrder() {
+        return order;
+    }
+
+    @Override
+    public String toString() {
+        return "WorkflowReorderActionStepForm{" +
+                "actionId='" + actionId + '\'' +
+                ", stepId='" + stepId + '\'' +
+                ", order=" + order +
+                '}';
+    }
+
+    public WorkflowReorderActionStepForm(final Builder builder) {
+
+        this.actionId           = builder.actionId;
+        this.stepId             = builder.stepId;
+        this.order              = builder.order;
+        this.checkValid();
+    }
+
+    public static final class Builder {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a constructor to the class, or provide default values. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1258)